packages/jest-snapshot/src/inline_snapshots.ts

```diff
@@ -176,6 +213,68 @@ const createParser = (
   return ast;
 };
 
+// This parser formats snapshots to the correct indentation.
+const createFormattingParser = (
+  inferredParser: string,
+  babelTraverse: Function,
+) => (
+  text: string,
+  parsers: {[key: string]: (text: string) => any},
+  options: any,
+) => {
+  // Workaround for https://github.com/prettier/prettier/issues/3150
+  options.parser = inferredParser;
+
+  const ast = getAst(parsers, inferredParser, text);
+  babelTraverse(ast, {
+    CallExpression({node: {arguments: args, callee}}: {node: CallExpression}) {
+      if (
+        callee.type !== 'MemberExpression' ||
+        callee.property.type !== 'Identifier' ||
+        callee.property.name !== 'toMatchInlineSnapshot' ||
+        !callee.loc
+      ) {
+        return;
+      }
+
+      let snapshotIndex: number | undefined;
+      let snapshot: string | undefined;
+      for (let i = 0; i < args.length; i++) {
+        const node = args[i];
+        if (node.type === 'TemplateLiteral') {
+          snapshotIndex = i;
+          snapshot = node.quasis[0].value.raw;
+        }
+      }
+      if (snapshot === undefined || snapshotIndex === undefined) {
+        return;
+      }
+
+      snapshot = indent(
+        snapshot,
+        Math.ceil(
+          !options.useTabs
```

Comment:
Maybe it's just me, but I find it harder to read with an additional negation - can we flip the ternary to avoid it?

Reply:
(same thing a few lines below)

Reply:
The reasoning for the current code is that I try to have the first part of an `if` resolve to the most common/expected path and then order the rest (any `if else`) in order of what's second-most-common, etc.

Since `pretty-format` is already assuming spaces and doesn't even support tabs at all, I wanted the primary logic to be spaces.

I can resolve both of our concerns by pulling the option out into a variable called `useSpaces`, though. What do you think?

Reply:
Sounds good 👍